packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -1969,31 +1957,42 @@ export class TeamsAppSolution implements Solution {
       }
 
       if (ctx.answers?.platform === Platform.CLI || ctx.answers?.platform === Platform.VSCode) {
+        const message = [
+          { content: `Listing M365 permissions\n`, color: Colors.BRIGHT_WHITE },
+          { content: `Account used to check: `, color: Colors.BRIGHT_WHITE },
+          { content: userInfo.userPrincipalName + "\n", color: Colors.BRIGHT_MAGENTA },
+          {
+            content: `Starting list all collaborators for environment: `,
```

Comment:
Starting list all teams app owners for environment

Reply:
Fixed. Thanks!